leptonai/photon/base.py

```diff
@@ -209,10 +217,14 @@ def find_local_photon(name: str) -> Optional[str]:
         )
     )
     record_or_none = res.fetchone()
+    logger.trace(f"find local photon: {record_or_none}")
     if record_or_none is None:
         return None
     else:
-        return record_or_none[0]
+        if return_path:
+            return record_or_none[3]
```

Comment:
@bddppq this is the essential line: returning [3] instead of [0]. An example record is below:

```
('/Users/jiayq/.cache/lepton/debug-by-lepton.1.photon', 'debug-by-lepton', 'hf:gpt2@11c5a3d5', '/Users/jiayq/.cache/lepton/debug-by-lepton.1.photon', 1704691038)
```